packages/frontend/components/Header.tsx

```diff
@@ -27,7 +53,83 @@ export default function Header() {
           smallScreen: "icon",
           largeScreen: "icon",
         }}
-      />
+      /> */}
+
+      {/* custom connect button */}
+      <ConnectButton.Custom>
+        {({ account, chain, openAccountModal, openChainModal, openConnectModal, authenticationStatus, mounted }) => {
+          // Note: If your app doesn't use authentication, you
+          // can remove all 'authenticationStatus' checks
```

Comment:
I noticed this custom button is coming from the rainbox kit examples: https://www.rainbowkit.com/docs/custom-connect-button

Could you remove the `authentication` part as they say? We are not going to use if for now.